### DISCUSSION TITLE
How is kia_uvo related to hyundai_kia_connect_api

### DISCUSSION THREAD

**Discussion by Bubo08** [Hyundai-Kia-Connect/kia_uvo#803] [Q&A]
I thought the hyundai_kia_connect_api is like a library and is used by kia_uvo. But checking the code I could not find out how they are connected.

**Comment by cdnninja** [comment]
Correct Kia_uvo is just a presentation layer. The manifest file specifies the version of the api that is used. Sensors etc all consume from vehicle class to display the data.  Vehicle manager is called to facilitate data updates.

**Reply by captain-ted** [reply]
Do you need to install both?

**Reply by cdnninja** [reply]
Kia_uvo will auto install and manage the api package.

**Comment by Bubo08** [comment]
Do you know in which directory the api library is placed in home assistant? I searched without success.

**Reply by cdnninja** [reply]
No sorry not sure.
